src/main/java/nl/topicus/mssql2monetdb/util/SerializableResultSetMetaData.java

```diff
@@ -0,0 +1,186 @@
+package nl.topicus.mssql2monetdb.util;
+
+import java.io.Serializable;
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.util.ArrayList;
+
+/**
+ * This class is used to persist a live ResultMetaData object to
+ * storage.
+ * 
+ * @author Dennis Pallett <dennis.pallett@topics.nl>
+ */
+public class SerializableResultSetMetaData implements ResultSetMetaData, Serializable {
+
+	private static final long serialVersionUID = 1L;
+	
+	private ArrayList<SerializableColumnMetaData> columnList;
+	
+	public SerializableResultSetMetaData (ResultSetMetaData liveMetaData) throws SQLException
+	{
+		
+		columnList = new ArrayList<SerializableColumnMetaData>(liveMetaData.getColumnCount());
+		
+		for (int i = 1; i <= liveMetaData.getColumnCount(); i++)
+		{
+			SerializableColumnMetaData col = new SerializableColumnMetaData();
+			
+			col.isAutoIncrement = liveMetaData.isAutoIncrement(i);
+			col.isCaseSensitive = liveMetaData.isCaseSensitive(i);
+			col.isSearchable = liveMetaData.isSearchable(i);
+			col.isNullable = liveMetaData.isNullable(i);
+			col.isSigned = liveMetaData.isSigned(i);
+			col.columnDisplaySize = liveMetaData.getColumnDisplaySize(i);
+			col.columnLabel = liveMetaData.getColumnLabel(i);
+			col.columnName = liveMetaData.getColumnName(i);
+			col.schemaName = liveMetaData.getSchemaName(i);
+			col.precision = liveMetaData.getPrecision(i);
+			col.scale = liveMetaData.getScale(i);
+			col.tableName = liveMetaData.getTableName(i);
+			col.catalogName = liveMetaData.getCatalogName(i);
+			col.columnType = liveMetaData.getColumnType(i);
+			col.columnTypeName = liveMetaData.getColumnTypeName(i);
+			col.isReadOnly = liveMetaData.isReadOnly(i);
+			col.isWritable = liveMetaData.isWritable(i);
+			col.isDefinitelyWritable = liveMetaData.isDefinitelyWritable(i);
+			col.columnClassName = liveMetaData.getColumnClassName(i);
+			
+			columnList.add(col);
+		}
+	}
+	
+	public SerializableResultSetMetaData ()
+	{
+		// empty
```

Comment:
hahaha, everyone can see that it is empty :P Maybe comment in why this is empty? :P